dist/images/ovndb-raft-functions.sh

```diff
@@ -17,24 +17,59 @@ verify-ovsdb-raft() {
   fi
 }
 
-# OVN DB must be up in the first DB node
-# This waits for ovnkube-db-0 POD to come up
-ready_to_join_cluster() {
+bracketify() { case "$1" in *:*) echo "[$1]" ;; *) echo "$1" ;; esac }
+
+# checks if a db pod is part of a current cluster
+db_part_of_cluster() {
+  local pod=${1}
+  local db=${2}
+  local port=${3}
+  echo "Checking if ${pod} is part of cluster"
+  init_ip=$(kubectl --server=${K8S_APISERVER} --token=${k8s_token} --certificate-authority=${K8S_CACERT} \
+    get pod -n ${ovn_kubernetes_namespace} ${pod} -o=jsonpath='{.status.podIP}')
+  if [[ $? != 0 ]]; then
+    echo "Unable to get ${pod} ip "
+    return 1
+  fi
+  echo "Found ${pod} ip: $init_ip"
+  init_ip=$(bracketify $init_ip)
+  target=$(ovn-${db}ctl --timeout=5 --db=${transport}:${init_ip}:${port} ${ovndb_ctl_ssl_opts} --data=bare --no-headings \
+    --columns=target list connection)
```

Comment:
L36: I don't think this will work unless you use `--no-leader-only`. Please see below:

```
# ovn-nbctl --timeout=5 --db=tcp:10.0.16.14:6641 --data=bare --no-headings     --columns=target list connection
ovn-nbctl: tcp:10.0.16.14:6641: database connection failed ()
# ovn-nbctl --no-leader-only --timeout=5 --db=tcp:10.0.16.14:6641 --data=bare --no-headings     --columns=target list connection
ptcp:6641
# 
```